test/parallel/test-internal-fs.js

```diff
@@ -8,4 +8,4 @@ const fs = require('internal/fs');
 assert.doesNotThrow(() => fs.assertEncoding());
 assert.doesNotThrow(() => fs.assertEncoding('utf8'));
 assert.throws(() =>
-    common.expectsError({code: 'ERR_INVALID_OPT_VALUE', type: TypeError}));
+  common.expectsError({code: 'ERR_INVALID_OPT_VALUE', type: TypeError}));
```

Comment:
Two comments:
1. you ate the test block `() => fs.assertEncoding('foo')`
2. common is undefined (required but not assigned)

This passed since `() => common.expectsError({code: 'ERR_INVALID_OPT_VALUE', type: TypeError}))` throw `Error Undefined`